### PR TITLE
Update definition of gist:Building. Fixes #482.

### LINKED ISSUE #482: Definition of gist:Building is too broad
Definition is "A man-made structure for dwelling or working."

According to this, construction scaffolds, parking lots (where a parking lot attendant works),  elevators, and food trucks, among many other things, are all buildings.

### PR DESCRIPTION
**Pull request by rjyounes**

### CODE REVIEW

File: gistCore.ttl [hunk 1]
```diff
@@ -186,7 +186,11 @@ gist:Building
 		gist:Artifact ,
 		gist:Landmark
 		;
-	skos:definition "A man-made structure for dwelling or working."^^xsd:string ;
+	skos:definition "A relatively permanent man-made structure situated on a plot of land, having a roof and walls, commonly used for dwelling, entertaining, or working."^^xsd:string ;
+	skos:example
```
Comment (uscholdm) [inline gistCore.ttl:190]: Maybe add yurt, which is a nice example of relatively permanant. Some are put up and taken down seasonally - but that still counts as relatively permanent, to me.

**Review by uscholdm — CHANGES_REQUESTED** [review]

**Comment by rjyounes** [comment]
I'm not sure why these other changes are showing up...

File: gistCore.ttl (hunk 1 above)
Comment (rjyounes) [reply gistCore.ttl:190]: I think yurt may be a controversial case, so I'm going to leave it out as subject to user interpretation.

**Comment by rjyounes** [comment]
I made a bad mistake - I submitted a PR to merge into master rather than develop. This is the result of working with another client that doesn't use a develop branch and merges directly to master, so I'm in the habit of doing that. I need to undo the merge to master and fix things again.

I also forgot release notes...

**Comment by rjyounes** [comment]
The merge to master has been undone.

**Pull request merged**